### PR TITLE
Austin code w/ zach and dr. jones changes

### PR DESCRIPTION
**Pull request by anw678**
my branch is named dr. jones code and i cant figure out how to change it but this is my code with all the changes implemented and some visual improvements that I asked about in teams but managed to figure out

### CODE REVIEW

File: anw678(TheDescent)/TheDescentPart2.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# The Descent
```
Comment (bjones1) [inline anw678(TheDescent)/TheDescentPart2.py]: Put your name in the title:

```
# *************************
# Austin W - The Descent
# *************************
```

File: anw678(TheDescent)/TheDescentPart2.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+while True:
+    max = 0
+    maxIndex = -1
+    for i in range(8):
+        # We will use "mountain_h" as our variable for the height of a mountain
+        mountain_h = int(input())
+#
+# Now we must write an action using print:
+#
+# The index of the mountain to fire on in the first test case is 4. To fire on this mountain we use:
```
Comment (bjones1) [inline anw678(TheDescent)/TheDescentPart2.py]: Use the following syntax to show code examples:

```
# .. code-block:: Python
#   :linenos:
#
#   print(4)
#
```

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by bjones1** [comment]
What do you want to change -- the branch's name?

File: anw678(TheDescent)/TheDescentPart2.py (hunk 2 above)
Comment (anw678) [reply anw678(TheDescent)/TheDescentPart2.py]: implemented throughout project. done

File: anw678(TheDescent)/TheDescentPart2.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+#
+# The Loop Solution
+# -----------------
+# The actual solution to this problem is to have the computer fire on the highest mountain like in our previous solution, but also allow the computer to take over our job of deciding which mountain is the highest.
+#
+# --------------------------------------
+#
+# The code will take the mountain height given by the game as an input, and compare it to the current highest mountain height stored in our variable "mountain_h". It will then output (or "fire on") the highest mountain so that the ship is safe for that turn of the game. This is repeated for 8 turns until the ship is safe to land.
+#
+# **Game Loop**
```
Comment (bjones1) [inline anw678(TheDescent)/TheDescentPart2.py:91]: This title is repeated -- did you mean to use something else here?

**Comment by bjones1** [comment]
Close enough -- thanks!

**Pull request merged**